refactor: change package setup to use vite

Sparkier opened the pull request:
This should make everything work with svelte kit and vite. In a demo svelte kit app I could use it without problems locally. If we want this update, I will make sure to update the sample to use sveltekit.

Sparkier commented:
For some reason, `yarn` needs to be run inside the storybook package now before building storybook. I have not had the time to investigate that yet.

domoritz commented:
@cabreraalex may know something since he's looked into sveltekit before.

cabreraalex commented:
Okay I updated storybook (got rid of TS because it was breaking and I got tired), and added a sveltekit example, just a copy of the svelte example.

We probably want to double check and test it separately before we send it and update. Or major version update?

domoritz commented on .eslintrc.js:
```diff
@@ -1,14 +1,10 @@
+	ignorePatterns: ['*.cjs'],
   env: {
-    es6: true,
+		es2017: true,
```
```suggestion
    es2017: true,
```

domoritz commented on packages/package.json:
What's this file for?

domoritz reviewed (approved):
Needs some prettification but otherwise looks good. I'm happy to do another major version.

Sparkier commented:
> Okay I updated storybook (got rid of TS because it was breaking and I got tired), and added a sveltekit example, just a copy of the svelte example.
> 
> We probably want to double check and test it separately before we send it and update. Or major version update?

I also have that implemented I think similarly in TS. Wanted to wait and see if this part is something we want to do before making another PR for the other things. Can post it as an alternative in another PR.

domoritz commented:
@cabreraalex can you make a final pass and merge + release?
